assets/css/main.css

```diff
@@ -11,20 +11,21 @@ body {
   text-align: center;
   max-width: 1200px;
   margin: 0 auto;
+  padding-top : 1em;
   padding-left: 1em;
   padding-right: 1em;
   display: flex;
   justify-content: center;
   flex-direction: column;
-  height: 400px;
+  min-height : 400px;
 }
 
 section{
   padding-top: 5px;
 }
 
 .header-logo{
-  width: 13%;
+  width: 40%;
```

Comment:
logo width seems too much. 25-30% seems okay and wouldn't need horizontal scroll.